src/main/resources/persistence.properties

```diff
@@ -4,7 +4,8 @@ hibernate.archive.autodetection = class
 
 javax.persistence.jdbc.url = jdbc:postgresql://188.166.92.29:5432/moodcat_db
 javax.persistence.jdbc.user = moodcat
-hibernate.hbm2ddl.auto = create-drop
+#hibernate.hbm2ddl.auto = create-drop
+hibernate.hbm2ddl.auto = update
```

Comment:
`git fetch origin && git rebase -i origin/master` and then change `pick` to `squash` for all the commits listed except the top one.